docs/en/maintain/backup.md

```diff
@@ -0,0 +1,202 @@
+# Backup and restore
+
+## High Availability Description
+
+* The nameserver cannot be fully hung up. Failover and recovery cannot be done if all hangs up
+* High availability is not guaranteed in the case of a single copy
+* In the case of two replicas and two nodes, the tablet can only hang on one node at most
```

Comment:
the tablet can only hang on one node at most => 
only one node of tablet can fail.